tests/python/test_transformation.py

```diff
@@ -81,7 +81,7 @@ def test_transformation():
     for key, val in rbc_s0.items():
         if not np.isclose(val, rbc_s[key], rtol=rtol, atol=atol):
             print(f"initial[{key}]={val}, final[{key}]={rbc_s[key]} not equal")
-        assert np.isclose(val, rbc_s[key], rtol=rtol, atol=atol)
+            assert np.isclose(val, rbc_s[key], rtol=rtol, atol=atol)
```

Comment:
Not sure this change is needed.
```suggestion
        assert np.isclose(val, rbc_s[key], rtol=rtol, atol=atol)
```

Reply:
Alternatively, we can just skip doing the `isclose` twice:
```suggestion
            assert False
```